modules/default/utils.js

```diff
@@ -17,20 +17,26 @@ async function performWebRequest (url, type = "json", useCorsProxy = false, requ
 		requestUrl = url;
 		request.headers = getHeadersToSend(requestHeaders);
 	}
-	const response = await fetch(requestUrl, request);
-	const data = await response.text();
 
-	if (type === "xml") {
-		return new DOMParser().parseFromString(data, "text/html");
-	} else {
-		if (!data || !data.length > 0) return undefined;
+	let response = await fetch(requestUrl, request);
```

Comment:
can still be a const, right?

```suggestion
	const response = await fetch(requestUrl, request);
```